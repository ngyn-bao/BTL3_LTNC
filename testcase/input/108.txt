INSERT x number
INSERT y string
BEGIN
INSERT x number
INSERT z number
BEGIN
INSERT t number
INSERT x number
RPRINT
END
END