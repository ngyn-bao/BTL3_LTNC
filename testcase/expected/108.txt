success
success
success
success
success
success
x//2 t//2 z//1 y//0